### PR TITLE
Tagging core graph utilities

### PR DESCRIPTION
**Pull request by aldebaran30701**
Refactor of anonymous plugins and tagging. Most of these plugins are tagged as VIEW as they change the camera or are camera related (perspective bookmarks)

### CODE REVIEW

File: CoreGraphUtilities/src/au/gov/asd/tac/constellation/graph/utilities/planes/PlaneManagerTopComponent.java [hunk 1]
```diff
@@ -627,4 +480,254 @@ private void setNode(final GraphNode node) {
...
+            wg.setObjectValue(planesAttr, 0, state);
+        }
+
+    }
+
+    /**
+     * Plugin to set the plane position on the graph.
+     */
+    @PluginInfo(pluginType = PluginType.VIEW, tags = {"VIEW"})
+    private static class SetPlanePositionPlugin extends SimpleEditPlugin {
```
Comment (arcturus2) [inline CoreGraphUtilities/src/au/gov/asd/tac/constellation/graph/utilities/planes/PlaneManagerTopComponent.java:615]: Hmm how about `MODIFY` ?

**Review by arcturus2 — CHANGES_REQUESTED** [review]
Some suggestions to spark a discussion. Noted that these are changing `META` attributes so I can understand where the `VIEW` came from too.

**Comment by aldebaran30701** [comment]
I've reviewed the tags and agree about your suggestions. I've kept the pluginTypes as VIEW because it seems to relate to the graph view.

**Review by antares1470 — CHANGES_REQUESTED** [review]

File: CoreGraphUtilities/src/au/gov/asd/tac/constellation/graph/utilities/perspectives/PerspectiveBookmarkTopComponent.java [hunk 2]
```diff
@@ -267,28 +243,16 @@ private void addButtonActionPerformed(java.awt.event.ActionEvent evt)//GEN-FIRST
...
-                    updateOnGraph(graph);
-                }
-            }).executeLater(graph);
+            Future<?> f = PluginExecution.withPlugin(new AddPerspectivePlugin(perspectiveModel, perspectivesList)).executeLater(graph);
+            try {
+                f.get();
+            } catch (InterruptedException ex) {
+                Thread.currentThread().interrupt();
+                Exceptions.printStackTrace(ex);
+            } catch (ExecutionException ex) {
```
[the remark this answers is not in the document]
Comment (aldebaran30701) [reply CoreGraphUtilities/src/au/gov/asd/tac/constellation/graph/utilities/perspectives/PerspectiveBookmarkTopComponent.java]: Fixed

**Review by antares1470 — APPROVED** [review]

**Review by arcturus2 — APPROVED** [review]

**Pull request merged**